_docs/advanced-features.md

```diff
@@ -215,11 +215,14 @@ eyJoaXN0b3J5IjpbMjA4MjI5MzQ4NV19
 ## Workflow Execution Service (WES) Command Line Interface (CLI)
 
 The Workflow Execution Service API describes a standard programmatic way to run and manage workflows. See more information here: 
-https://github.com/ga4gh/workflow-execution-service-schemas
+[https://github.com/ga4gh/workflow-execution-service-schemas](https://github.com/ga4gh/workflow-execution-service-schemas).
 
 The Dockstore CLI implements a WES client that allows users to submit a request to launch a workflow run, get the status of a run, or cancel a
 run at a WES endpoint. 
 
+Workflow inputs must be described so that the remote executor can access them. In practice, this means providing them using HTTP URLs in the 
+workflow input file. In addition, to download outputs, the workflow and executor must be configured to make these data available using HTTP methods.
```

Comment:
Can this be generalized to anything remote? (e.g. s3:// ftp://, etc.)

Reply:
It depends on the WES endpoint. Arvados supports at least https and http; Cromwell supports https, http and if configured properly gs://.

Reply:
Yeah, I decided to write HTTP because its the minimum requirement. I don't think we want to try to keep our docs up to date with all those specific details of the other software.